manim/mobject/types/vectorized_mobject.py

```diff
@@ -2022,6 +2022,43 @@ def __init__(self, vmobject, **kwargs):
 
 
 class DashedVMobject(VMobject, metaclass=ConvertToOpenGL):
+    """
+
+    Examples
+    --------
+    .. manim:: DashedVMobjectExample
+        :save_last_frame:
+
+        class DashedVMobjectExample(Scene):
+            def construct(self):
+                r = 0.5
+                pos = 2.5 * UP + 6 * LEFT  # Tracker for position on screen
+                # Top Row: Increases the number of dashes per iteration
+                for dashes in range(2, 12):  # [2 , ..., 11]
```

Comment:
Echoing @kolibril13's  comment: spelling out what the range produces doesn't seem necessary.
```suggestion
                for dashes in range(2, 12): 
```